user_dist : 0.0
user_time : 0.0
user_finish : 0.0
type : 0
points : [(0.0,0.0)]